docs/source/building/platforms/maxwell_desy.rst

```diff
@@ -5,11 +5,11 @@ This page only provides HiPACE++ specific instructions.
 For more information please visit the
 `Maxwell documentation <https://confluence.desy.de/display/MXW/Maxwell+Cluster>`__.
 
-Create a file ``profile.hipace`` and ``source`` it whenever you log in and want to work with
+Create a file ``profile.hipace``, for instance in ``$HOME``, and ``source`` it whenever you log in and want to work with
 HiPACE++:
 
 .. code-block:: bash
-
+   #!/usr/bin/env zsh # Shell is assumed to be zsh
```

Comment:
would you need here also
```
module ()
{
eval `modulecmd bash $*`
}
```

Reply:
If I source profile.hipace directly from the console it is not necessary, but if the source is in another script that is executed in a new shell (e.g. bash script.sh or ./script.sh) then it is needed.

Reply:
I see, I added this as a top

Reply:
When I view this file here https://github.com/Hi-PACE/hipace/blob/5cdd01ac330a7a846771edb0fd21bdf69f813314/docs/source/building/platforms/maxwell_desy.rst , the module load stuff doesn't show up.